tests/vanilla/basic.test.tsx

```diff
@@ -2,6 +2,8 @@ import { expect, it } from 'vitest'
 import { atom } from 'jotai/vanilla'
 
 it('creates atoms', () => {
+  // primitive atom without initial value
+  const undefinedAtom = atom()
```

Comment:
I'm sorry, but I didn't mean this test.
Please revert it and add "type" tests in `tests/vanilla/types.test.tsx`.